assets/css/components/button.scss

```
@@ -0,0 +1,114 @@
+.btn {
+  @extend %font-ui-text-base-good;
+  color: $color-white-160;
+
+  &:hover {
+    color: $color-white-180;
+  }
+
+  &:focus {
+    box-sizing: border-box;
+    box-shadow: 1px $color-focus;
+  }
+
+  &:disabled {
+    color: $color-white-160;
+  }
+
+  &--sm {
+    @extend %font-ui-text-base-good;
```

```suggestion
    @extend %font-ui-text-base-sm-good;
```
This should be different from the default `.btn`